### PR TITLE
test: deal upd with new spell

### PR DESCRIPTION
**Pull request by vasd85**

### CODE REVIEW

File: src/lib/helpers/utils.ts [hunk 1]
```diff
@@ -54,7 +54,7 @@ export function stringifyUnknown(unknown: unknown): string {
     }
 
     if (unknown instanceof Error) {
-      return jsonStringify(unknown, Object.getOwnPropertyNames(unknown));
+      return `${unknown.message}\n${unknown.stack ?? ""}`;
     }
```
Comment (akim-bow) [inline src/lib/helpers/utils.ts:58]: What is the reason for this change?
For example Error object could have more properties than just `message` and `stacktrace`

File: test/const.ts
Comment (akim-bow) [inline test/const.ts:1]: I would suggest renaming `const.ts` -> `constants.ts`.
Sounds more familiar and clearer.
Up to you.

File: src/lib/helpers/utils.ts (hunk 1 above)
Comment (shamsartem) [reply src/lib/helpers/utils.ts]: The reason is that when the error object is stringified like that it's rendered completely unreadable

I decided improve that according to your recommendation so other properties can be still visible, take a look

**Review by shamsartem — APPROVED** [review]
